src/nupic/algorithms/Svm.hpp

```diff
@@ -905,10 +908,10 @@ namespace nupic {
 		   int cache_size =100,
 		   int shrinking =1,
 		   bool probability =false,
-		   int seed =-1)
+		   int seed =0)
 	  : param_(kernel, probability, gamma, C, eps, cache_size, shrinking),
 	    problem_(new problem_type(n_dims, true, threshold)),
-	    model_(nullptr), rng_(seed != -1 ? seed : 0),
+	    model_(nullptr), rng_(seed != RANDOM_SEED ? seed : 0),
```

Comment:
I'd say just change the default in Random class but it takes a UInt so it would be a complete interface change. This is a bit problematic and the standard C++ library also takes an unsigned int (srand) so I wouldn't want to change it for ours. Let's kick this conversation back to the issue to find a resolution.